benchmarking/evaluations/metrics/priority_satisfaction.py

```diff
@@ -11,10 +11,18 @@ class PrioritySatisfaction(TeamSetMetric):
     Calculates the number of priorities satisfied by the algorithm
     """
 
-    def __init__(self, priorities: List[Priority], is_linear: bool, *args, **kwargs):
+    def __init__(
+        self,
+        priorities: List[Priority],
+        is_linear: bool,
+        exponent_base: int = 2,
```

Comment:
nit;

uhhhh, how does this affect existing code? Feel like it should default to whatever the actual prio algorithm is using, like import NUM_BRACKETS

Reply:
This is what it was before. It was not the same as num buckets previously.

Reply:
Let me know if this is fine

Reply:
O